src/controllers/verify-ketl.ts

```diff
@@ -66,7 +72,7 @@ export default class VerifyKetlController {
         const attestationHash = message[1]
         secret.push(attestationHash)
       }
-      secret.push(`${type}${signature}`)
+      secret.push(`a${type}${signature}`)
```

Comment:
The signature is a set of numbers, it would be convenient to add some separator like a letter before the attestation type